src/types/index.ts

```diff
@@ -34,3 +37,20 @@ export interface NuxtSecurityRouteRules {
   allowedMethodsRestricter?: AllowedHTTPMethods | false;
   nonce?: boolean;
 }
+
+export interface NuxtSecurityEventContext {
+  headers: Record<string, string | string[] | number | false> | null
```

Comment:
Should this share the types of `SecurityHeaders`?

Reply:
oops indeed